docs/markdown/Rust-module.md

```diff
@@ -34,7 +34,7 @@ that automatically.
 Additional, test only dependencies may be passed via the dependencies
 argument.
 
-### bindgen(*, input: string | BuildTarget | []string | []BuildTarget, output: string, include_directories: []include_directories, c_args: []string, args: []string)
+### bindgen(*, input: string | BuildTarget | [](string | BuildTarget), output: string, include_directories: []include_directories, c_args: []string, args: []string)
```

Comment:
This is not actually solving any kind of confusion, the href anchors and section headers are the wrong place to use ad-hoc, complex syntax.

Still, this is hardly worse than before.

And the other two changes seem correct.

Reply:
This isn't really ad-hoc, it's typescript syntax, and I was using this long before we added the python style syntax of the yaml.  Like many things in meson this has all grown rather organically. (I really prefer this to the `List[str]` syntax anyway, since Meson only has arrays and dicts anyway, and it's less wordy.